classes/converter.php

```diff
@@ -42,18 +42,26 @@ class converter implements \core_files\converter_interface {
     /** @var array $imports List of supported import file formats */
     private static $imports = [
         'doc' => 'application/msword',
+        'docm' => 'application/vnd.ms-word.document.macroEnabled.12 ',
```

Comment:
Extra space at the end. Is this intentional?

Reply:
Thanks @keevan,

It have missed this one, there should be no space. I have updated the MR

Reply:
Hi @keevan ,

Would you  please have another review on this MR?

Thanks

Reply:
I have removed "macro enabled" extension, as there is potential security issue